include/tesla.hpp

```diff
@@ -2357,30 +2357,28 @@ namespace tsl {
              * @param onValue Value drawn if the toggle is on
              * @param offValue Value drawn if the toggle is off
              */
-            StepTrackBar(const char icon[3], std::initializer_list<std::string> stepDescriptions) 
-                : TrackBar(icon), m_numSteps(stepDescriptions.size()), m_stepDescriptions(stepDescriptions.begin(), stepDescriptions.end()) { }
+            StepTrackBar(const char icon[3], size_t count)
```

Comment:
Call this `numSteps` like the member variable. Count doesn't really describe what it does